script/pump.py

```diff
@@ -62,12 +62,12 @@
        EXPRESSION has Python syntax.
 """
 
-__author__ = 'wan@google.com (Zhanyong Wan)'
-
 import os
 import re
 import sys
 
+__author__ = 'wan@google.com (Zhanyong Wan)'
```

Comment:
Do we use this file?

Reply:
It generates `native_mate/native_mate/constructor.h`